tests/unit_tests/cli/test_integration_cli.py

```diff
@@ -436,3 +436,60 @@ def test_that_prior_is_not_overwritten_in_ensemble_experiment(
                 pd.testing.assert_frame_equal(parameter_values, prior_values)
         else:
             pd.testing.assert_frame_equal(parameter_values, prior_values)
+
+
+def test_field_init_file_not_readable(copy_case, monkeypatch):
+    monkeypatch.setattr(
+        ensemble_evaluator._wait_for_evaluator, "WAIT_FOR_EVALUATOR_TIMEOUT", 5
+    )
+    copy_case("snake_oil_field")
+    config_file_name = "snake_oil_field.ert"
+    field_file_rel_path = "fields/permx0.grdecl"
+    os.chmod(field_file_rel_path, 0x0)
+    parser = ArgumentParser(prog="test_field_init_segfault")
+    parsed = ert_parser(
+        parser,
+        [
+            TEST_RUN_MODE,
+            config_file_name,
+        ],
+    )
+
+    try:
+        run_cli(parsed)
+    except ErtCliError as err:
+        assert "failed to open" in str(err)
```

Comment:
Should probably add to the test that it did not fail to close the file.

Reply:
hm do you mean that we should that it did not throw an exception because of troubles with closing the file? isn't that implicitly covered by checking that "open" is in the error message?